transport/model/project_member.go

```diff
@@ -0,0 +1,16 @@
+package model
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type ProjectMember struct {
+	UserID      uuid.UUID `json:"user_id"`
+	Name        string    `json:"name"`
+	Email       string    `json:"email"`
+	ProjectRole string    `json:"project_role"`
+	CreatedAt   time.Time `json:"created_at"`
+	UpdatedAt   time.Time `json:"updated_at"`
+}
```

Comment:
The response in the transport layer can be simplified to look like this. It returns all the information needed regarding the project member.

```
{
        "user_id": "3bb77a43-bdf4-4659-a74b-9aa451a28aa1",
        "name": "John",
        "email":"test@test.tt"
        "project_role": "editor",
        "created_at": "2024-02-01T09:00:00+00:00",
        "updated_at": "2024-04-27T08:00:00+00:00"
}
```